### PR TITLE
updating to support cloud9 dev environment

### PR DESCRIPTION
**Pull request by brentley**
Adds a theme script, and changes the port to 8080 (to support c9 dev environment)

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,30 +1,31 @@
...
+    "url": "https://github.com/brentley/eksworkshop/issues"
+  },
+  "license": "ISC",
+  "main": "index.js",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/brentley/eksworkshop.git"
+  },
+  "scripts": {
+    "theme": "git submodule init && git submodule update",
```
Comment (brentley) [inline package.json]: This is the addition to initialize the theme submodule:
`npm run theme`

File: package.json [hunk 2]
```diff
@@ -1,30 +1,31 @@
...
+    "drafts": "hugo server -w -v -DF --disableFastRender --enableGitInfo --bind=0.0.0.0",
+    "server": "hugo server -w -v --enableGitInfo --bind=0.0.0.0 --port 8080 --navigateToChanged",
```
Comment (brentley) [inline package.json]: This change adds on the specific port 8080 because that's what cloud9 requires.

**Comment by brentley** [comment]
no idea why the whole file looks like a diff/change -- I notated the only 2 changes.

File: package.json (hunk 1 above)
Comment (rnzsgh) [reply package.json]: Should these links point to the aws-samples repo?

**Comment by brentley** [comment]
Aah, these don't really do anything, but can be changed for consistency.

**Pull request merged**